scripts/exglobal_diag.sh

```diff
@@ -47,7 +47,7 @@ export NLN=${NLN:-"/bin/ln -sf"}
 export CHGRP_CMD=${CHGRP_CMD:-"chgrp ${group_name:-rstprod}"}
 export NEMSIOGET=${NEMSIOGET:-${NWPROD}/exec/nemsio_get}
 export NCLEN=${NCLEN:-$HOMEgfs/ush/getncdimlen}
-export CATEXEC=${CATEXEC:-$HOMEgfs/exec/ncdiag_cat.x}
+export CATEXEC=${CATEXEC:-$HOMEgfs/exec/nc_diag_cat.x}
```

Comment:
This is not the correct name for ncdiag_cat.x.  The serial ncdiag_cat.x should be ncdiag_cat_serial.x.

Reply:
Okay, when I build I also get a `nc_diag_cat.x`, so I thought that was the correct one. I will fix.

Reply:
Thanks for addressing this.  The GSI creates a parallel and serial executable for `ncdiag_cat`.  Before the update, `ncdiag_cat.x` was for the serial executable, while `ncdiag_cat_mpi.x` was the parallel version.  With the update, the parallel version became `nc_diag_cat.x` and the serial is `ncdiag_cat_serial.x`.